submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,177 @@
+const NUM_OF_FRIENDS = 20;
+const FRIENDS = document.querySelector(".friends");
+const MALE_RADIO_BUTTON = document.getElementById("male");
+const FEMALE_RADIO_BUTTON = document.getElementById("female");
+const ALL_RADIO_BUTTON = document.getElementById("all");
+let sortedFriends;
+let allFriends;
+let isFiltered = false;
+
+function fetchFriends() {
+  const apiUrl = `https://randomuser.me/api/?results=${NUM_OF_FRIENDS}`;
+  fetch(apiUrl)
+    .then(handleErrors)
+    .then((response) => response.json())
+    .then((data) => (allFriends = data.results))
+    .then(() => addFriends(allFriends))
+    .catch(showErrorMessage);
+}
+
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+function showErrorMessage() {
+  const wholePage = document.body;
+  wholePage.innerHTML = `<div class="errorMessage">Ops...Something went wrong.</div>`;
+}
+
+function addFriends(friendsToBeAdded) {
+  let fragment = document.createDocumentFragment();
+  friendsToBeAdded.forEach((friend) => {
+    let friendCard = document.createElement("div");
+    friendCard.setAttribute("class", "friend_card");
+    friendCard.innerHTML = `<div class="card_wrapper ${friend.gender}"><div class="name ">${`${friend.name.first} ${friend.name.last}`}</div>
+                                <div class="photo"><img src="${friend.picture.large}"></div>
+                                <div class="info_block">
+                                <div class="age">Age ${friend.dob.age}</div>
+                                <div class="place">${friend.location.city}</div>
+                                <div class="email"><a href="mailto:${friend.mail}" class="email_link">${friend.email}</a></div>
+                                <div class="gender">${friend.gender}</div>
+                                </div></div>
+                                `;
```

Comment:
Like this.

```suggestion
friendCard.innerHTML = `
    <div class="card_wrapper ${friend.gender}">
        <div class="name ">${`${friend.name.first} ${friend.name.last}`}</div>
        <div class="photo">
           <img src="${friend.picture.large}">
         </div>
        <div class="info_block">
            <div class="age">Age ${friend.dob.age}</div>
            <div class="place">${friend.location.city}</div>
            <div class="email">
                <a href="mailto:${friend.mail}" class="email_link">${friend.email}</a>
            </div>
            <div class="gender">${friend.gender}</div>
        </div>
    </div>
`;
```

Also, you should not use `div` for everything ;)